packages/react/src/utilities/ThemeProvider/__snapshots__/ThemeProvider.test.tsx.snap

```diff
@@ -13,7 +13,15 @@ exports[`ThemeProvider can apply body theme to body 1`] = `
         font-size: 14px;
         font-weight: 400;
       }
-/>
+>
+  <div>
+    <div
+      class="foo"
+    >
+      app
+    </div>
+  </div>
+</body>
```

Comment:
Is this change expected?

Reply:
yes it is, that's the proper DOM tree rendered, unlikely with enzyme which for whatever reason emmited only `<body />` which is incorrect